main.cpp

```diff
@@ -242,7 +242,9 @@ int main (int argc, char * const argv[]) {
 	FLDBHeader header;
 	if(!header.Read(f)) {
 		printf("ERROR: '%s' is not an FLDB file\n", db_filename);
-		goto done_close;
+    	fclose(f);
+	    f = nullptr;
```

Comment:
If it's not too much to ask, could you fix the indentation for this line and the next? Thx!

Reply:
Sure. I guess one of my editors still has tab in place of spaces...